pages/_app.tsx

```diff
@@ -1,20 +1,27 @@
+import React from 'react';
 import Link from 'next/link';
+
+import ViewportProvider from 'components/organisms/ViewportProvider';
+import '../styles/globals.css';
+
 import { PrismicProvider } from '@prismicio/react';
```

Comment:
Ain't your linter warning you about import order? I vaguely remember seeing it a week ago

Reply:
it does, but it seems here it doesn't.

Reply:
yes it does, heavens bless it. It says ViewportProvider should be ABOVE the prismic stuff. By the way there's a chance the former will be killed